Lib/test/test_pprint.py

```diff
@@ -241,6 +249,9 @@ def test_same_as_repr(self):
                              .replace('\n', ' '), native)
             self.assertEqual(pprint.pformat(simple, underscore_numbers=True), native)
             self.assertEqual(pprint.saferepr(simple), native)
+            with contextlib.redirect_stdout(None):
+                # smoke test - there is no output to check
+                pprint.pprint(simple)
```

Comment:
Is not it already tested in `test_stdout_is_None`?

Reply:
You're right, this is no longer needed, I'll remove. With my first (more complex) solution this covered more code paths.